service/src/main/java/tds/config/services/impl/ConfigServiceImpl.java

```diff
@@ -6,9 +6,11 @@
 import org.springframework.stereotype.Service;
 import tds.config.ClientSystemFlag;
 import tds.config.ClientTestProperty;
+import tds.config.TimeLimits;
 import tds.config.repositories.ConfigRepository;
 import tds.config.services.ConfigService;
 
+import java.sql.Time;
```

Comment:
Is this used?  And if so, should we be using a java version instead of a SQL version?